### PR TITLE
Automated OCP-25777

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei please help to review. will give pass log later. Thanks

### CODE REVIEW

File: lib/rules/web/admin_console/4.2/deployment.xyaml [hunk 1]
```diff
@@ -88,3 +88,16 @@ check_current_cpu_utilization:
   element:
     selector:
       <<: *value
+check_pod_status:
+  elements:
+  - selector: &status
+      xpath: //tr/td[contains(.,'<pod_name>')]/following-sibling::td[2][contains(.,'<pod_status>')]
```
Comment (yapei) [inline lib/rules/web/admin_console/4.2/deployment.xyaml:94]: I think we don't need to add index here, just to make sure the string is at the same row with pod name

File: lib/rules/web/admin_console/4.5/deployment.xyaml [hunk 2]
```diff
@@ -96,4 +96,15 @@ check_current_cpu_utilization:
   element:
     selector:
       <<: *value
+check_pod_status:
+  elements:
+  - selector: &status
+      xpath: //tr/td[contains(.,'<pod_name>')]/following-sibling::td[2][contains(.,'<pod_status>')]
+    timeout: 120
```
Comment (yapei) [inline lib/rules/web/admin_console/4.5/deployment.xyaml:103]: Do we need so long timeout here?

**Comment by SSshahan** [comment]
I think so. 120 is more safier.

File: lib/rules/web/admin_console/4.2/deployment.xyaml (hunk 1 above)
Comment (SSshahan) [reply lib/rules/web/admin_console/4.2/deployment.xyaml:94]: will update

File: lib/rules/web/admin_console/4.5/deployment.xyaml (hunk 2 above)
Comment (SSshahan) [reply lib/rules/web/admin_console/4.5/deployment.xyaml:103]: Yeah. We need to wait the pod changed from containerCreating to ImagePullBackOff  on console. https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/84416/artifact/embedded_files/2020-04-13T06%3A15%3A19+00%3A00-screenshot.png

**Comment by yapei** [comment]
Thanks for the updates

File: lib/rules/web/admin_console/4.1/deployment.xyaml [hunk 3]
```diff
@@ -88,3 +88,4 @@ check_current_cpu_utilization:
   element:
     selector:
       <<: *value
+
```
Comment (liangxia) [inline lib/rules/web/admin_console/4.1/deployment.xyaml:91]: What's this change for ?

File: testdata/pods/two_container_sleep.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```
Comment (liangxia) [inline testdata/pods/two_container_sleep.yaml:1]: Where is this file used ?

Comment (SSshahan) [reply testdata/pods/two_container_sleep.yaml:1]: https://github.com/openshift/cucushift/pull/7577

File: lib/rules/web/admin_console/4.1/deployment.xyaml (hunk 3 above)
Comment (SSshahan) [reply lib/rules/web/admin_console/4.1/deployment.xyaml]: updated

File: testdata/pods/two_container_sleep.yaml (hunk 4 above)
Comment (liangxia) [reply testdata/pods/two_container_sleep.yaml:1]: Then move the test file to cucushift, along with the scenario.

Comment (SSshahan) [reply testdata/pods/two_container_sleep.yaml]: updated, please help to merge

**Pull request merged**